framework/components/ACheckbox/ACheckbox.js

```diff
@@ -254,10 +258,12 @@ ACheckbox.propTypes = {
    */
   wrap: PropTypes.bool,
   /**
-   * Size options ["default","medium","small"]
-   * Default size is 15px, Magnetic Medium = 20px, Magnetic Small = 16px
+   * Option for small size (default is medium)
    */
-  size: PropTypes.oneOf(["default", "small", "medium"])
+  small: PropTypes.bool,
+  /**
+   * ** Deprecated ** Use small prop for sizing down (medium is default)
+   */ size: PropTypes.oneOf(["default", "small", "medium"])
```

Comment:
This might be a linter issue, but there should be a line break here.

```suggestion
   */
   size: PropTypes.oneOf(["default", "small", "medium"])
```